assets/alertmanager/alertmanager.yaml

```diff
@@ -89,10 +90,44 @@ spec:
       name: secret-alertmanager-kube-rbac-proxy
     - mountPath: /etc/tls/private
       name: secret-alertmanager-main-tls
+    - mountPath: /etc/tls/client
+      name: metrics-client-ca
+      readOnly: false
+  - args:
+    - --secure-listen-address=0.0.0.0:9097
+    - --upstream=http://127.0.0.1:9096
+    - --config-file=/etc/kube-rbac-proxy/config.yaml
+    - --tls-cert-file=/etc/tls/private/tls.crt
+    - --tls-private-key-file=/etc/tls/private/tls.key
+    - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
```

Comment:
might have to synchronize with https://github.com/openshift/cluster-monitoring-operator/pull/1409